.eleventy.js

```diff
@@ -3,7 +3,6 @@ const eleventyNavigationPlugin = require("@11ty/eleventy-navigation");
 const pluginSyntaxHighlight = require("@11ty/eleventy-plugin-syntaxhighlight");
 const eleventyPluginTOC = require("eleventy-plugin-nesting-toc");
 
-const slugify = require("slugify");
```

Comment:
1️⃣ Eleventy uses `slugify` by default now

Reply:
Is this the slugify that includes these special characters? I remember that being the reason we had to initially include it.